pom.xml

```diff
@@ -207,6 +207,11 @@
                 <artifactId>testng</artifactId>
                 <version>${testng.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-lang3</artifactId>
+                <version>${org.apache.commons.version}</version>
+            </dependency>
```

Comment:
SInce this dependency is used for test classes, it would be nice if we can set the scope to test here. WDYT?